### DISCUSSION TITLE
补药关啊！

### DISCUSSION THREAD

**Discussion by Yizakl** [share121/inter-knot#5503] [公告]
![IMG_0170](https://github.com/user-attachments/assets/919ddcbe-f86c-40b3-8fc6-d61ce1e8caf2)
![IMG_0171](https://github.com/user-attachments/assets/de63fdb2-d82e-4888-87d9-045e7ac25b65)

**Comment by KELERSAMA** [comment]
大佬

**Reply by Yizakl** [reply]
你才是大佬喵～

**Reply by KELERSAMA** [reply]
我是fvv喵

**Comment by wzyaeu** [comment]
阿米诺斯
